core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/MigrationUtils.scala

```diff
@@ -96,9 +117,43 @@ object MigrationUtils {
     }
 
 
-    implicit class ToJsonOps(data: Map[String, Any]) {
-      def toJson: JsValue = JsObject(data.view.mapValues(s => new JsString(s.toString)).toMap)
+    implicit class ToJsonOps(data: Iterable[(String, Any)]) {
+      def toJson: JsValue = JsObject(
+        data.map(Arrow[Function].second((s: Any) => new JsString(s.toString))).toMap
+      )
     }
+
+    implicit def monadErrorDBIOAction[E <: Effect]
```

Comment:
Noting this also exists :) https://github.com/RMSone/slick-cats

Reply:
It does! But only defines for DBIO rather than the DBIOAction trait

Reply:
really interesting stuff. thanks for the explanation!